lib/src/test/java/com/auth0/jwt/JWTCreatorTest.java

```diff
@@ -431,4 +436,209 @@ public void shouldAcceptCustomArrayClaimOfTypeLong() throws Exception {
         String[] parts = jwt.split("\\.");
         assertThat(parts[1], is("eyJuYW1lIjpbMSwyLDNdfQ"));
     }
-}
+    
+    @Test
+    public void shouldAcceptCustomClaimOfTypeObject() throws Exception {
```

Comment:
This is testing the "with map" setter. Let's rename it a bit.
```suggestion
    public void shouldAcceptCustomClaimOfTypeMap() throws Exception {
```